src/main/java/com/minecolonies/coremod/colony/Colony.java

```diff
@@ -1073,14 +1094,19 @@ public void onWorldTick(@NotNull final TickEvent.WorldTickEvent event)
             //  Spawn Citizens
             if (townHall != null && citizens.size() < maxCitizens)
             {
-                int respawnInterval = Configurations.citizenRespawnInterval * 20;
+                int respawnInterval = Configurations.gameplay.citizenRespawnInterval * 20;
                 respawnInterval -= (60 * townHall.getBuildingLevel());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 60 to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)